### PR TITLE
Updated version number for wpfLib and ZeroTouchlib nuget pkg. 

### PR DESCRIPTION
**Pull request by rk-lakshmanan**
### Purpose

This PR is to update the version number WPFLib and ZeroTouchLib nuget packages.
Included DynamoUnits in ZeroTouch package as well.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.

### Reviewers

@sharadkjaiswal

### CODE REVIEW

File: tools/NuGet/BuildDynamoZeroTouchLibraryPackage.bat [hunk 1]
```diff
@@ -15,5 +15,6 @@ mkdir .\%zt%\tools
 
 copy %base%\ProtoGeometry.dll .\%zt%\lib\%framework%\ProtoGeometry.dll
 copy %base%\ProtoGeometry.xml .\%zt%\build\%framework%\ProtoGeometry.xml
+copy %base%\DynamoUnits.dll .\%zt%\lib\%framework%\DynamoUnits.dll
```
Comment (sharadkjaiswal) [inline tools/NuGet/BuildDynamoZeroTouchLibraryPackage.bat]: Do we have DynamoUnits.xml, that can go to build folder?

File: tools/NuGet/DynamoVisualProgramming.ZeroTouchLibrary/ZeroTouchLibrary.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>DynamoVisualProgramming.ZeroTouchLibrary</id>
-    <version>1.0.0-beta2</version>
+    <version>1.0.0-beta3</version>
```
Comment (sharadkjaiswal) [inline tools/NuGet/DynamoVisualProgramming.ZeroTouchLibrary/ZeroTouchLibrary.nuspec]: Can we update the description to say:
```
It contains following files:
   1. DynamoServices.dll
```

Comment (sharadkjaiswal) [inline tools/NuGet/DynamoVisualProgramming.ZeroTouchLibrary/ZeroTouchLibrary.nuspec]: What if we make this dependent on DynamoServices version 1.0.0?? Will it still work?

File: tools/NuGet/DynamoVisualProgramming.Core/Core.nuspec [hunk 3]
```diff
@@ -2,19 +2,25 @@
...
+		3)DynamoShapeManager.dll 
+		4)DynamoUtilities.dll
+	</description>
     <summary>This package contains the core assemblies for Dynamo.</summary>
     <copyright>Copyright Autodesk 2016</copyright>
     <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
     <dependencies>
         <dependency id="Prism" version="4.1"/>
         <dependency id="NUnit" version="2.6.3"/>
+		<dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="1.0.0-beta3"/>
```
Comment (aparajit-pratap) [inline tools/NuGet/DynamoVisualProgramming.Core/Core.nuspec]: @sharadkjaiswal why do we need for Core to have a dependency on ZeroTouchLibrary?

**Pull request merged**